app/models/hidden_status.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: hidden_statuses
+#
+#  id         :bigint(8)        not null, primary key
+#  account_id :bigint(8)        not null
+#  status_id  :bigint(8)        not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class HiddenStatus < ApplicationRecord
+  update_index('statuses#status', :status) if Chewy.enabled?
```

Comment:
This is probably not needed, as you don't want a hidden status to be added to the full-text search index for that account.